packages/frontend/src/hooks/useProcessQr.ts

```diff
@@ -284,20 +282,24 @@ export default function useProcessQR() {
       }
       /**
        * DCACCOUNT:
+       * contains the url to a chatmail relay
        * see https://github.com/deltachat/interface/blob/main/uri-schemes.md#DCACCOUNT
+       * or
+       *
+       * DCLOGIN
+       * complete login credentials for a relay or mail server
+       * see https://github.com/deltachat/interface/blob/main/uri-schemes.md#DCLOGIN
        *
-       * contains the url to a chatmail instance which will be used in the
-       * instant onboarding process initiated by this QR code scan
        *
        * Scenarios depending on the account status:
-       * 1. configured account: Ask the user if they want to create a new profile on the given chatmail instance
-       * 2. unconfigured account: set instant onboarding chatmail instance
+       * 1. configured account: Ask the user if they want to add a new relay to the current profile
+       * 2. unconfigured account: set instant onboarding instance
```

Comment:
```suggestion
       * 2. unconfigured account: set instant onboarding chatmail instance
```

What's the reason for removing "Chatmail"? Can it be something other than Chatmail?